packages/workbox-core/src/utils/assert.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * A simple assertion class to use for development builds.
+ *
+ * This will be excluded from production builds.
+ */
+class Assertions {
+
+}
+
+export default new Assertions();
```

Comment:
So do you want exported functions, a class with static functions or a static instance of the class?

I've just switched to exporting the class itself and then workbox core instantiating, but I feel like you want something different.

For me, I just view this as a singleton (i.e. why would we ever want more than a single instance of this library? The plus of this being a class is that it makes it easier to ensure with it's being exported in the final output.